packages/tools/pluggable-widgets-tools/configs/webpack.native.config.js

```diff
@@ -95,9 +95,9 @@ const widgetConfig = {
         new CopyWebpackPlugin({
             patterns: [
                 {
-                    from: `${variables.projectPath}/src/**/*.xml`,
+                    from: path.posix.join(path.resolve(variables.projectPath, "src").replace(/\\/g, "/"), "**/*.xml"),
```

Comment:
I think we prefer destructuring node dependencies nowadays

Reply:
Done